xbmc/cores/RetroPlayer/playback/GameLoop.h

```diff
@@ -54,15 +55,15 @@ class CGameLoop : protected CThread
   void Process() override;
 
 private:
-  double FrameTimeMs() const;
-  double SleepTimeMs() const;
-  double NowMs() const;
+  std::chrono::microseconds FrameTimeUs() const;
+  std::chrono::microseconds inline SleepTimeUs() const;
+  std::chrono::microseconds NowUs() const;
 
   IGameLoopCallback* const m_callback;
   const double m_fps;
   std::atomic<double> m_speedFactor;
-  double m_lastFrameMs = 0.0;
-  mutable double m_adjustTime = 0.0;
+  std::chrono::microseconds m_lastFrameUs = std::chrono::microseconds::zero();
+  mutable std::chrono::microseconds m_adjustTime = std::chrono::microseconds::zero();
```

Comment:
Small request, can you brance initialiuze (`std::chrono::microseconds m_lastFrameUs{std::chrono::microseconds::zero()};`), and also be so kind as the kill the mutable - the function that moidifies this shouldn't be const.

Reply:
Yes, I will make those changes. I won't be able to get to it until tomorrow.

Reply:
I know I said the function that modifies the variable shouldn't be const because of the mutable variable, but instead the function should be const like its neighbors _and_ there should be no mutable variable. Can you refactor so that SleepTimeUs() is const?

Reply:
I just removed it as it is no longer required.